<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>path</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string> 文件</string>
    </property>
    <addaction name="actionOpen_new_window"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="actionFormat"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>系统表</string>
    </property>
    <addaction name="actionShowTable"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUp"/>
   <addaction name="actionLarge"/>
   <addaction name="actionSmall"/>
   <addaction name="actionHome"/>
   <addaction name="actionLogin"/>
   <addaction name="actionAddUser"/>
   <addaction name="actionUsersInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_new_window">
   <property name="text">
    <string>open new window</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actionFormat">
   <property name="text">
    <string>format</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="icon">
    <iconset>
     <normaloff>up.jpg</normaloff>up.jpg</iconset>
   </property>
   <property name="text">
    <string>up</string>
   </property>
  </action>
  <action name="actionLarge">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/large.jpg</normaloff>:/new/prefix1/large.jpg</iconset>
   </property>
   <property name="text">
    <string>large</string>
   </property>
  </action>
  <action name="actionSmall">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/small.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>small</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/home.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>home</string>
   </property>
  </action>
  <action name="actionShowTable">
   <property name="text">
    <string>showTable</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/person.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </action>
  <action name="actionAddUser">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/addUser.jpg</normaloff>:/new/prefix1/addUser.jpg</iconset>
   </property>
   <property name="text">
    <string>addUser</string>
   </property>
  </action>
  <action name="actionUsersInfo">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/persons.jpg</normaloff>:/new/prefix1/persons.jpg</iconset>
   </property>
   <property name="text">
    <string>usersInfo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
